zebrad/src/components/inbound.rs

```diff
@@ -352,18 +352,24 @@ impl Service<zn::Request> for Inbound {
                     })
                 .boxed()
             }
-            zn::Request::PushTransaction(_transaction) => {
-                debug!("ignoring unimplemented request");
-                // TODO: send to Tx Download & Verify Stream
-                // https://github.com/ZcashFoundation/zebra/issues/2692
+            zn::Request::PushTransaction(transaction) => {
+                if let Setup::Initialized { tx_downloads, .. } = &mut self.network_setup {
+                    // TODO: check if we're close to the tip before proceeding?
+                    // what do we do if it's not?
```

Comment:
We might need some way to inspect `mempool` or check if near tip, if mempool is 'activated' (because it's near tip or otherwise) we would proceed handling this, otherwise just ignore it as before. For now, punting that handling to `download_if_needed_and_verify` is ok.

Reply:
After https://github.com/ZcashFoundation/zebra/pull/2741 that can be handled inside the mempool service itself. I think this can be implemented as part of https://github.com/ZcashFoundation/zebra/issues/2629

Reply:
I removed the `TODO` comments from here as we are not doing this check in this part of the code.

https://github.com/ZcashFoundation/zebra/pull/2727/commits/7f287b7243ac1ba8f51714872d47d2c8f0449fd4